docs/index.bs

```diff
@@ -1241,6 +1257,13 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
         "all"
       };
     </pre>
+    <pre class="idl" id="client-state-enum">
+      enum ClientState {
+        "active",
+        "frozen",
+        "all"
+      };
+    </pre>
```

Comment:
I think we should break this out into multiple enums, since a client can never have a state of "all".

I think enums can inherit, so ClientStateQuery (which adds "all") can inherit from ClientState.

Reply:
I can't get enums to inherit.. I don't see any that do that in [WPT](https://github.com/web-platform-tests/wpt/tree/master/interfaces) . Either way added a separate definition